ajax/libs/plotly.js/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "plotly.js",
+  "version": "1.30.0",
+  "description": "The open source javascript graphing library that powers plotly",
+  "filename": "plotly.min.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/plotly/plotly.js.git"
+  },
+  "author": "Plotly, Inc.",
+  "homepage": "https://plot.ly/javascript/",
+  "keywords": [
+    "graphing",
+    "plotting",
+    "data",
+    "visualization",
+    "plotly"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/plotly/plotly.js.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "plotly!(*with-meta)*.!(md|json)"
```

Comment:
Please use `**/plotly!(*with-meta)*.!(md|json)` because its `basePath` is `dist`. Sorry for noticing this previously...